libopae/plugins/xfpga/sysfs.c

```diff
@@ -307,29 +307,29 @@ STATIC int sysfs_region_destroy(sysfs_fpga_region *region)
 int sysfs_region_count(void)
 {
 	int res = 0;
-	if (pthread_mutex_lock(&_sysfs_region_lock)) {
-		FPGA_MSG("Error locking sysfs region lock");
-	} else {
+	if (!opae_mutex_lock(res, &_sysfs_region_lock)) {
 		res = _sysfs_region_count;
```

Comment:
You'll need a separate variable to capture the count, because res will be reset on 314.

Reply:
Just testing you